api/src/features/multiflash/multiflash_use_case.py

```diff
@@ -7,15 +7,25 @@
 
 
 class MultiflashResponse(BaseModel):
-    ph_index: List[str] = Field(..., description="Phase labels (vapor, liquid, aqueous, mercury)")
-    ph_frac: List[float] = Field(..., description="Fraction of each phase (sums to 1)")
+    phase_label: List[str] = Field(..., description="Phase labels (vapor, liquid, aqueous, mercury)")
```

Comment:
This could be stronger typed. Ether list of PhaseEnum. or a list of literals

Reply:
Good idea. Can make an enum.